string soma_roi_id
string type
int32 roi_knowledge
int32[] roi_knowledge_hourly
